test/index.spec.js

```diff
@@ -57,15 +57,18 @@ describe('@blackbaud/skyux-builder', () => {
     });
   });
 
-  it('should handle unknown command', () => {
+  it('should return false unknown command', () => {
```

Comment:
The name of this test should probably be "should return false _for_ unknown command".

Reply:
"for" sure.  :-)  Updated.

Reply:
Forget the "for"? :)